libs/service-portal/law-and-order/src/lib/const.ts

```diff
@@ -0,0 +1,43 @@
+import { LawAndOrderDefenseChoiceEnum } from '@island.is/api/schema'
+
+interface Choice {
+  code: LawAndOrderDefenseChoiceEnum
+  message: {
+    id: string
+    defaultMessage: string
+  }
+}
+
+// Get localized messages for defense choices in Subpoena
+export const DefenseChoices: Record<string, Choice> = {
+  WAIVE: {
+    code: LawAndOrderDefenseChoiceEnum.WAIVE,
+    message: {
+      id: 'sp.law-and-order:no-defender',
+      defaultMessage: 'Ég óska ekki eftir verjanda',
+    },
+  },
+  CHOOSE: {
+    code: LawAndOrderDefenseChoiceEnum.CHOOSE,
+    message: {
+      id: 'sp.law-and-order:choosing-lawyer',
+      defaultMessage:
+        'Ég óska þess að valinn lögmaður verði skipaður verjandi minn',
+    },
+  },
+  DELAY: {
+    code: LawAndOrderDefenseChoiceEnum.DELAY,
+    message: {
+      id: 'sp.law-and-order:delay-choice',
+      defaultMessage:
+        'Ég óska eftir fresti fram að þingfestingu til þess að tilnefna verjanda',
+    },
+  },
+  DELEGATE: {
+    code: LawAndOrderDefenseChoiceEnum.DELEGATE,
+    message: {
+      id: 'sp.law-and-order:choose-for-me',
+      defaultMessage: 'Ég fel dómara málsins að tilnefna og skipa mér verjanda',
+    },
+  },
+}
```

Comment:
Think you should do this in the service, make the frontend dumber

Reply:
Yes good idea 💯